gax/test/test-application/src/index.ts

```diff
@@ -378,6 +378,11 @@ async function testEchoError(client: EchoClient) {
 }
 
 async function testEchoErrorWithRetries(client: SequenceServiceClient) {
+  // TODO: This test needs to be fixed as createSequenceRequestFactory sends
+  //  back four UNAVAILABLE errors, but only an UNAVAILABLE error and a
+  //  DEADLINE_EXCEEDED error are encountered. The test needs to be corrected
+  //  later on to expect two UNAVAILABLE errors, but we'll keep it as is now
+  //  to unblock the CI pipeline.
```

Comment:
I am not comfortable merging this with a broken test

Reply:
Ok. We can spend more time to make sure the test is correct, but the underlying reason the test produces the wrong result was there before the PR was in place. I decided to write technical debt to get this change through, but we can push through to make everything correct if necessary.

Reply:
There are certain kinds of tech debt exceptions I am willing to be more flexible on but unfortunately with this one, we really can't properly test your stuff. I am trying to submit a PR to update showcase to see if that automagically fixes things

Reply:
I'll revisit these changes since it looks like that DEADLINE_EXCEEDED error has been properly addressed.